.github/welcome_bot.yml

```diff
@@ -0,0 +1,28 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+Hello there! :wave: Welcome to the project! :sparkling_heart:
+Thankyou and congrats :tada: for opening your very first issue in this project. AnitaB.org is an inclusive community,committed to creating a safe and positive environment.
+:cherry_blossom: Please adhere to our [Code of Conduct](https://github.com/anitab-org/mentorship-android/blob/develop/.github/CODE_OF_CONDUCT.md) . :raised_hands:
+You may submit a PR on the same if you like! If you want to report a bug :beetle: please follow our [bug_report.md](https://github.com/anitab-org/.github/blob/main/.github/ISSUE_TEMPLATE/bug_report.md) template.
```

Comment:
@blacrabb update the bug report link to this https://github.com/anitab-org/.github/blob/main/.github/ISSUE_TEMPLATE/bug_report.md
and this is good to merge :)